src/controller/python/ChipDeviceController-ClusterCommands.cpp

```diff
@@ -0,0 +1,1002 @@
+} // namespace Identify
+
+namespace LevelControl {
+void HandleMoveCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+void HandleMoveToLevelCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+void HandleMoveToLevelWithOnOffCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+void HandleMoveWithOnOffCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+void HandleStepCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+void HandleStepWithOnOffCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+void HandleStopCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+void HandleStopWithOnOffCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+} // namespace LevelControl
+
+namespace OnOff {
+void HandleOffCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+void HandleOnCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+void HandleToggleCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+} // namespace OnOff
+
+namespace Scenes {
+void HandleAddSceneCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+void HandleGetSceneMembershipCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+void HandleRecallSceneCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+void HandleRemoveAllScenesCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+void HandleRemoveSceneCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+void HandleStoreSceneCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+void HandleViewSceneCommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+} // namespace Scenes
+
+namespace TemperatureMeasurement {
+} // namespace TemperatureMeasurement
+
+} // namespace cluster
+} // namespace app
+} // namespace chip
+
+extern "C" {
+
+// Cluster BarrierControl
+CHIP_ERROR chip_ime_AppendCommand_BarrierControl_BarrierControlGoToPercent(chip::Controller::Device * device,
+                                                                           chip::EndpointId ZCLendpointId, chip::GroupId ZCLgroupId,
+                                                                           uint8_t percentOpen)
+{
+    if (device == nullptr)
+    {
+        return CHIP_ERROR_INCORRECT_STATE;
+    }
+    chip::Controller::BarrierControlCluster clusterBarrierControl;
+    (void) ZCLgroupId;
+    clusterBarrierControl.Associate(device, ZCLendpointId);
+    return clusterBarrierControl.BarrierControlGoToPercent(nullptr, percentOpen);
```

Comment:
For what it worth. I'm going to have very similar code to call commands in `chip-tool` once I will be done with a local patch to use `ChipCluster` API instead of the raw API from `src/app/encoder.cpp`. There is probably some duplication we can avoid here. I should have a PR with my changes tomorrow. I would see if I can do something that makes our 2 use cases happy.